zhaquirks/__init__.py

```diff
@@ -33,6 +33,25 @@ def __init__(self, *args, **kwargs):
 class LocalDataCluster(CustomCluster):
     """Cluster meant to prevent remote calls."""
 
+    async def bind(self):
+        """Bind cluster."""
+        return (foundation.Status.SUCCESS,)
+
+    async def unbind(self):
+        """Unbind cluster."""
+        return (foundation.Status.SUCCESS,)
+
+    async def configure_reporting(
+        self,
+        attribute,
+        min_interval,
+        max_interval,
+        reportable_change,
+        manufacturer=None,
+    ):
+        """Configure reporting."""
+        return foundation.Status.SUCCESS
```

Comment:
While this would work, because currently we don't really check the response, it may break once we start doing that. Instead, return proper `ConfigureReportingResponse`, eg 
```suggestion
        return foundationConfigureReportingResponse.deserialize(b"\x00)[0]
```

Reply:
Thanks! I didn't have an idea we had `ConfigureReportingResponse`!